apps/fluent-tester/src/FluentTester/FluentTester.tsx

```diff
@@ -87,11 +87,16 @@ export const FluentTester: React.FunctionComponent<FluentTesterProps> = (props:
 
     return (
       <View style={fluentTesterStyles.header}>
-        <Text style={[fluentTesterStyles.testHeader]} variant="heroLargeSemibold" color={theme.host.palette?.TextEmphasis}>
+        <Text
+          testID={BASE_TESTPAGE}
```

Comment:
Uhhh this made the Text size of the header tiny in win32. Is this fixed now?

Reply:
I just made the change just to see if it would help w/ passing the uwp e2e.